packages/hotpotqa/src/aviary/envs/hotpotqa/env.py

```diff
@@ -186,6 +197,13 @@ def __init__(
         self.incorrect_reward = incorrect_reward
         self.tool_failure_reward = tool_failure_reward
         self.proxy = proxy
+        self.evaluation_mode = evaluation_mode
+
+        if evaluation_mode == EvalAnswerMode.LLM_SCORE:
+            raise ValueError(
+                f'{HotPotQAEnv.__name__} does not support "{evaluation_mode}" '
+                "since it requires binary evaluation of the answer."
+            )
```

Comment:
```suggestion
            raise NotImplementedError(
                f'{HotPotQAEnv.__name__} does not support "{evaluation_mode}"'
                " since the environment was built around binary evaluation of the"
                " answer. Further development is needed for this mode."
            )
```